com.blueokiris.bgrm.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "com.blueokiris.bgrm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bgrm-fp.sh",
+    "finish-args": [
+        "--device=all",
+        "--socket=wayland",
+        "--socket=x11",
```

Comment:
does it really support wayland? I yes, then it's `fallback-x11`

also missing share=ipc

Reply:
It does not support wayland afaik.
I removed that and added the share ipc